plugins/hls-eval-plugin/test/Main.hs

```diff
@@ -140,13 +140,8 @@ tests =
   , goldenWithEval "Can handle eval inside nested comment properly" "TNested" "hs"
   , goldenWithEval "Test on last line insert results correctly" "TLastLine" "hs"
   , testGroup "with preprocessors"
-    [ knownBrokenInEnv [HostOS Windows, GhcVer GHC86]
```

Comment:
You were a little too zealous - we still need to say it's broken on Windows.

Reply:
Assumed it was `and` not `or` 😅 Good that there is CI for all cases.